protocol/messenger_images.go

```diff
@@ -1,7 +1,5 @@
 package protocol
 
-import "fmt"
-
 func (m *Messenger) ImageServerURL() string {
-	return fmt.Sprintf("https://localhost:%d/messages/", m.httpServer.Port)
+	return m.httpServer.MakeImageServerURL()
```

Comment:
URLs are the concern of `Server`, `Messenger` doesn't want to know how they were made